dev/src/codewind/project/Project.ts

```diff
@@ -464,6 +466,77 @@ export default class Project implements vscode.QuickPickItem {
         });
     }
 
+    public async onLoadRunnerUpdate(event: {projectID: string, status: string, timestamp: string}): Promise<void> {
+        Log.d(`${this.name} load runner status changed to ${event.status}`, event);
+        switch (event.status) {
+            case "hcdReady":
+            case "profilingReady":
+                Log.d("Profiling data is ready to be saved to workspace");
+                const loadTestPath = path.join(this.localPath.fsPath, "load-test");
+                const timestampPath = path.join(loadTestPath, event.timestamp);
+                let fileName = "";
+                if (this.language === "java") {
```

Comment:
The languages are in constants in `ProjectType.Languages`, use those. I also suggest using `this.language.toLowerCase()` just in case the user input `Java` instead of `java`.